library/std/src/keyword_docs.rs

```diff
@@ -102,7 +102,9 @@ mod break_keyword {}
 
 #[doc(keyword = "const")]
 //
-/// Compile-time constants and deterministic functions.
+/// Compile-time constants and compile-time evaluable functions.
```

Comment:
TIL 'evaluable' means 'able to be evaluated'! https://english.stackexchange.com/questions/304948/evaluable-vs-evaluatable